app/src/main/java/com/example/dkirilova/gym/fragments/AvailabilityDetailsFragment.java

```diff
@@ -34,37 +34,37 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             public void onClick(View v) {
 
                 String dayName = etDayName.getText().toString().trim();
-                int startTime = 0;
-                int duration = 0;
+                int startTime;
+                int duration;
```

Comment:
Suggestion: it is good practice to have all variables that allow it - final
this would also help identify bugs, assignments that are problematic and is in general less error-prone